st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -44,14 +45,20 @@ def __init__(self):
         self._coordinator = coordination.get_coordinator()
         self.get_one_db_method = self.__get_by_name
 
-    @jsexpose(arg_types=[str])
-    def get_one(self, name):
+    @jsexpose(arg_types=[str, str])
+    def get_one(self, name, decrypt='false'):
```

Comment:
Can we hook this up and make bool work so we don't need to abuse string type?

I thought we already do this somewhere, but maybe we don't...